src/Campaign/Customer.php

```diff
@@ -164,6 +164,16 @@ public function login(array $credentials): self
         return new self($this->api()->get($this->getEndpoint('login'), $credentials));
     }
 
+    /**
+     * @param array $parameters
+     *
+     * @return \Buzz\Control\Campaign\Customer
```

Comment:
Should this return annotation indicate a string rather than Customer model?

Reply:
Ah, looking at this yes, it will return the property->value which is a string. Have updated!